crates/compiler/build/src/program.rs

```diff
@@ -166,7 +166,7 @@ fn gen_from_mono_module_llvm<'a>(
 
     let kind_id = Attribute::get_named_enum_kind_id("alwaysinline");
     debug_assert!(kind_id > 0);
-    let enum_attr = context.create_enum_attribute(kind_id, 1);
+    let enum_attr = context.create_enum_attribute(kind_id, 0);
```

Comment:
Why does this need to change? Can we please lift the 0 to a constant? Here and below.

Reply:
well it needs to change because in llvm a value of `1` just does not work, and causes weird LLVM issues (the attribute is just invalid and llvm verification fails). 

so, this is not a constant, the value has no meaning, and `0` just turns out to be ignored by llvm 15 (and 16). So I don't know what a constant would add here.

Reply:
also, this broke because it was never tested. I've opened a PR on inkwell to test these function attributes